src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const CENTURY_DIVIDER = 100;
+  const filteredPeople = people.filter(person =>
+    (century !== undefined
+    && Math.ceil(person.died / CENTURY_DIVIDER) === century
+    && person.sex === 'm')
```

Comment:
```suggestion
    && isMan(person))
```
Consider creating a helper function for conducting this check